### PR TITLE
✨ Added newsletter design settings

### PR DESCRIPTION
**Pull request by peterzimon**
no refs.
Adds settings for email newsletters

### CODE REVIEW

**Review by rshbhgrg — APPROVED** [review]

File: core/server/data/schema/default-settings.json [hunk 1]
```diff
@@ -383,5 +383,46 @@
...
+            "validations": {
+                "isEmpty": false,
+                "isIn": [
+                    [
+                        "serif",
+                        "sans_serif"
+                    ]
+                ]
+            },
+            "type": "boolean"
```
Comment (daniellockyer) [inline core/server/data/schema/default-settings.json:425]: `s/boolean/string` 😛

File: core/server/data/migrations/versions/3.38/06-add-newsletter-settings.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+const logging = require('../../../../../shared/logging');
+
+module.exports = {
```
Comment (daniellockyer) [inline core/server/data/migrations/versions/3.38/06-add-newsletter-settings.js:3]: Could/should we try and use `createTransactionalMigration` here?

**Review by daniellockyer — CHANGES_REQUESTED** [review]

File: core/server/data/schema/default-settings.json (hunk 1 above)
Comment (rishabhgrg) [reply core/server/data/schema/default-settings.json:425]: 😅 Damn!

File: core/server/data/migrations/versions/3.38/06-add-newsletter-settings.js (hunk 2 above)
Comment (rishabhgrg) [reply core/server/data/migrations/versions/3.38/06-add-newsletter-settings.js]: Done!

**Review by daniellockyer — APPROVED** [review]

**Review by allouis — APPROVED** [review]

**Pull request merged**